src/internal/utils/date-time/join-date-time.ts

```diff
@@ -4,3 +4,8 @@
 export function joinDateTime(dateString: string, timeString: string) {
   return `${dateString}T${timeString}`;
 }
+
+export function splitDateTime(dateStr: string) {
```

Comment:
Taking suggestions how to rename the file, since now it includes both ways, split and join

Reply:
Since we seem to be splitting every function into its own file, this could also go into `slit-date-time.ts`

Reply:
I would prefer having symmetric functions in a single file (because they are always changed together)